documentation/modules/exploit/multi/http/october_upload_bypass_exec.md

```diff
@@ -0,0 +1,45 @@
+
+## Description
+An authenticated user with permission to upload and manage media contents can
+upload various files on the server. The application prevents the user from
+uploading PHP code by checking the file extension. It uses black-list based
+approach, as seen in octobercms/vendor/october/rain/src/Filesystem/
+Definitions.php:blockedExtensions().
+
+## Vulnerable  Sorftware
+https://www.exploit-db.com/apps/4ff8a9688f31b7338020d0bc85da13fc-october-1.0.412.tar.gz
```

Comment:
The `Vulnerable Software` section should make reference to the vulnerable version (`v1.0.412 (build 412)`), and whether previous versions were affected.

Reply:
Handled this before landing.